content/blog/2022/eddy-symposium-report/index.md

```diff
@@ -0,0 +1,44 @@
+
+## Context
+
+The Symposium was framed by the interesting and interdisciplinary scientific career of the late [Jack Eddy](https://en.wikipedia.org/wiki/John_A._Eddy). Eddy's legacy was highlighted and his influence has been extended by dynamic leadership from NASA Program Officer [Madhulika Guhathakurta (Lika)](https://science.nasa.gov/about-us/organization-and-leadership/lead-program-scientist-for-lws). Lika helped launch and has sustained NASA's [Living with a Star (LWS)](https://science.nasa.gov/heliophysics/programs/living-with-a-star) program over the past two decades. Prior to LWS, NASA had a variety of siloed efforts focused on near-Sun and near-Earth behavior. The LWS program led to an integration of these efforts under "system science" or "systems engineering" approaches and an expressed desired to connect LWS research activities with impacts on Earth (society, biology, culture, etc.). The program has expanded to include explorations of similar questions arising around other planets in our solar system and the recently discovered collection of exoplanets. Scientists from diverse disciplines (plasma physics, stellar evolution, atmospheric chemistry, space weather, planetary science,...) work together on "cross disciplinary" research that helps us understand our lives near our star.
+
+
+The Symposium focused on three disciplinary areas (**Exoplanets**; **Sun-Climate and Star-Climate interactions**; **Risk and resilience of space weather**) unified under the cross-cutting thread of **open science**. Frequent references were made to the upcoming 2023 Year of Open Science and [NASA's Transform to Open Science (TOPS) mission](https://science.nasa.gov/open-science/transform-to-open-science). Symposium attendees listened to talks surveying the four areas in the morning and participated in hackathon-style breakout projects during the afternoons. Work on the projects launched at the Eddy Symposium [continues](https://github.com/jack-eddy-symposium)...
+
+## 2i2c's role
+
+2i2c, with input from Symposium CoChair's [Dan Marsh](https://staff.ucar.edu/users/marsh) and [Ryan McGranaghan](http://www.ryanmcgranaghan.com/about.html), rapidly deployed a cloud-hosted JupyterHub for use during the event. The work to set up that hub, openly chronicled in [this GitHub issue](https://github.com/2i2c-org/infrastructure/issues/1329), included swapping out a Pangeo-style software environment for a heliophysics-specific resource developed by [HelioCloud](http://heliocloud.org/) with special thanks to [Brian Thomas](https://www.nasa.gov/offices/oce/bios/mwm/Thomas-Brian-bio.html)!
```

Comment:
Maybe we should include a one-sentence description of the purpose of the hub? Something like `The hub provided a shared space for participants to explore data, run analyses, and collaborate with one another`.